recipe/meta.yaml

```diff
@@ -48,8 +48,6 @@ test:
         {% endfor %}
         - test -d $PREFIX/include/flann                 # [not win]
         - if not exist %PREFIX%\\Library\\include\\flann exit 1      # [win]
-        - conda inspect linkages -p $PREFIX flann       # [not win]
```

Comment:
was there a problem with running these?

Reply:
These lines were deleted by the bot in https://github.com/conda-forge/flann-feedstock/pull/30/commits/36267f6d03e0eefd0d5009075d98873b12d4d0d6 - I guess they're deprecated but I'm not sure.

Reply:
ok! perfect